src/alfanous-django/templates/base.html

```diff
@@ -121,6 +121,24 @@
 
     </style>
 
+<script type="application/ld+json">
+{
+  "@context": "http://schema.org",
+  "@type": "Organization",
```

Comment:
Isn't http://schema.org/SoftwareApplication better to describe Alfanous as name for search engines? Organization is not our mean product or job.

Reply:
Yes, may be